providers/databricks/tests/system/databricks/example_databricks_sensors.py

```diff
@@ -30,6 +31,8 @@
 # [DAG name to be shown on Airflow UI]
 DAG_ID = "example_databricks_sensor"
 
+WAREHOUSE_ID = os.environ.get("WAREHOUSE_ID", "")  # Defaults to an empty string to satisfy typing
```

Comment:
It's best to throw an exception when it's empty...

Reply:
warehouse_id is a required param for the APIs to work. I guess we have mentioned those as positional params but if it receives empty values I would expect the APIs to fail and report an error.

But yes, like mentioned by @alexott might make sense to raise an error from the operator/sensor if we get an empty warehouse_id